### PR TITLE
Fix select query

### PR DESCRIPTION
**Pull request by VadimDyachenko**
connect #82

### CODE REVIEW

File: src/main/java/juja/microservices/users/dao/crm/domain/UserCRM.java [hunk 1]
```diff
@@ -63,4 +69,6 @@ public UserCRM() {
 | c_slack            | varchar(255)     | YES  |     | NULL    |                |
 | c_isStudent        | tinyint(1)       | NO   |     | 0       |                |
 | c_uuid             | varchar(255)     | YES  |     | NULL    |                |
+| assignedTo         | varchar(50)      |      |     |         |                |
+| visibility         | int(11)          |      |     |         |                |
```
Comment (OlgaKulikova) [inline src/main/java/juja/microservices/users/dao/crm/domain/UserCRM.java]: what about null and default attributes?

Comment (OlgaKulikova) [reply src/main/java/juja/microservices/users/dao/crm/domain/UserCRM.java]: @danilkuznetsov Your turn!

Comment (adaleorion) [reply src/main/java/juja/microservices/users/dao/crm/domain/UserCRM.java]: @VadimDyachenko 

| visibility         | int(11)          | NO   |     | NULL    |                |
| assignedTo         | varchar(50)      | YES  | MUL | NULL    |                |

Comment (VadimDyachenko) [reply src/main/java/juja/microservices/users/dao/crm/domain/UserCRM.java]: Done

**Review by adaleorion — APPROVED** [review]

**Review by adaleorion — APPROVED** [review]
:+1:

**Pull request merged**